### DISCUSSION TITLE
Plugin repository is inaccessible

### DISCUSSION THREAD

**Discussion by Nodrev** [apache/netbeans#5322] [General]
Hello,

I had installed Netbeans (both 8.2 and 12.x) on a new dev machine, and I need to install the legacy c++ plugin.
I have enabled the "legacy 8.2 plugin repository", but the repository seem to be down, and thus, I cannot get the plugin...
The strange thing is that the "main" plugin repository seem to be down too, as I cannot access to https://plugins.netbeans.apache.org/ neither 
Is there a problem with those servers (I did not saw any announcement about this)?

**Comment by mbien** [comment]
servers are indeed having some issues at the moment. It is being worked on.
